task/buildah/0.1/buildah.yaml

```diff
@@ -185,6 +193,10 @@ spec:
         BUILDAH_ARGS+=("--target=${TARGET_STAGE}")
       fi
 
+      if [ -n "${BUILD_ARGS_FILE}" ]; then
+        BUILDAH_ARGS+=("--build-arg-file=${BUILD_ARGS_FILE}")
```

Comment:
Does the path need to be local to the context, i.e. similar to the Dockerfile?
```
$SOURCE_CODE_DIR/$CONTEXT/$DOCKERFILE
```

Reply:
yep,

fixed